app/src/main/java/org/mozilla/fenix/settings/logins/fragment/LoginDetailFragment.kt

```diff
@@ -219,7 +219,15 @@ class LoginDetailFragment : SecureFragment(R.layout.fragment_login_detail) {
     ) : View.OnClickListener {
         override fun onClick(view: View) {
             val clipboard = view.context.components.clipboardHandler
-            clipboard.text = value
+            when (snackbarText) {
+                R.string.logins_username_copied -> {
+                    clipboard.text = value
+                }
+
+                R.string.logins_password_copied -> {
+                    clipboard.sensitiveText = value
+                }
+            }
             showCopiedSnackbar(view.context.getString(snackbarText))
```

Comment:
[According to the docs](https://developer.android.com/develop/ui/views/touch-and-input/copy-paste#duplicate-notifications) we should avoid presenting our own notification that something has been copied to the clipboard. While we're in here we should try to clean this up as well :)